src/h3/mod.rs

```diff
@@ -476,6 +480,20 @@ impl Config {
     pub fn set_qpack_blocked_streams(&mut self, v: u64) {
         self.qpack_blocked_streams = Some(v);
     }
+
+    /// Sets the `poll()` repetition threshold for DATAGRAM events.
```

Comment:
We might want to go into a bit more detail on what these config options do, and why one might want to use them, as it's not immediately evident IMO.